x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/resource_findings_table.test.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import * as TEST_SUBJECTS from '../../test_subjects';
+import { ResourceFindingsTable, ResourceFindingsTableProps } from './resource_findings_table';
+import { TestProvider } from '../../../../test/test_provider';
+
+import Chance from 'chance';
+import { CspFinding } from '../../../../../common/schemas/csp_finding';
+import { EcsEvent } from '@kbn/ecs';
+import { capitalize } from 'lodash';
+import moment from 'moment';
+
+const chance = new Chance();
+
+const getFakeFindings = (): CspFinding & { id: string } => ({
```

Comment:
this can prove useful for the team, please move it to `x-pack/plugins/cloud_security_posture/public/test/fixtures` instead :)